src/api.js

```diff
@@ -3,9 +3,15 @@ import axios from "axios";
 const api = axios.create({
   baseURL: "https://myas-news-api.onrender.com/api",
 });
+
 export const getArticles = () => {
   return api("/articles").then(({ data }) => {
-    console.log(data);
+    return data;
+  });
+};
+
+export const getArticle = (article_id, title) => {
+  return api(`/articles/${article_id}`).then(({ data }) => {
```

Comment:
you are passing through an argument of `title` and declaring a parameter that isn't being used in the function, best practice would be to remove this for better readability and improved performance